instagram/src/Components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import './PostContainer.css';
+import CommentSection from "../CommentSection/CommentSection";
+
+class PostContainer extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            likes: 0,
+            noLike: 'far fa-heart',
+            like: 'hidden',
+        }
+    }
+    componentDidMount() {
+        this.setState(state => ({
+            likes: this.props.likes,
+        }))
+    }
+
+    likeHandler = () => {
+        this.setState(state => ({
+            likes: this.state.likes + 1,
+            noLike: 'hidden',
+            like: 'fas fa-heart',
+        }))
+    }
+
+    dislikeHandler = () => {
+        this.setState(state => ({
+            likes: this.state.likes - 1,
+            noLike: 'far fa-heart',
+            like: 'hidden',
+        }))
+    }
+
+    render() {
+        return (
+            <div className="post-container">
+                {/* Header Container */}
+                <div className="header-container">
+                    {/* User image and username */}
+                    <img src={this.props.userImg} alt="User" className="user-img"/>
+                    <p className="username">{this.props.username}</p>
+                </div>
+                {/* Post image*/}
+                <img src={this.props.postImg} alt="Post" className="post-img"/>
+                {/* Like and Comment Icons + Llke Counter */}
+                <div className="post-interaction">
+                    {/* Icons */}
+                    <div className="post-icons">
```

Comment:
Nice job getting like and dislike to work with fontawesome. Good methods for them and functionality.